faqrenderer/faq.md

```diff
@@ -0,0 +1,121 @@
+
+## For Instructors
+
+- How do I distribute files to students?
+- How do I set up custom environments for my class?
+
+
+## Which environment should I choose?
+
+After logging in to JupyterHub, you will be taken to a Server Options page where you may select your JupyterLab evironment. This will determine the languages, packages, and kernels available to you. You should continue with whichever option is selected by default unless you have a specific reason to do otherwise.
+
+The New Default Environment is what most users should choose for general JupyterLab use. It is an updated version of what is now called Legacy Default Environment. In this new environment, all supported languages except for Octave and C++ have been upgraded. Depending on the language, some syntax might have changed, so be sure to test your code to see if it works on a newer version. Here's a list of all the languages we support and the version changes:
+
+| Language | Old version | New version |
+| - | - | - |
+| Python | 3.6 | 3.7.8 |
+| R | 3.6.1 | 4.0.2 |
+| Julia | 1.1.0 | 1.5.0 |
+| Octave | 4.2.2 | 4.2.2 |
+| C++ (cling) | 0.6 | 0.6 |
+| SageMath | 8.1 | 9.1 |
+
+Most of the packages included in this New Default Environment are at their latest version as of August 2020. Compared to the legacy environment, a few packages have been removed in this new environment, which are listed below. 
+
+- pymol and ipymol: Requires extra conda channels and is not free software. Can be installed manually in a new conda environment.
+- opty: Requires Python <=3.6. Can be installed manually in a new conda environment with Python 3.6.
+- rpy2: Requires R <= 4.0. Can be installed manually in a new conda environemt with R 3.6.
+
+If you require the above packages or older versions of existing ones, you should use the legacy environment or [create a conda environment](#how-do-i-create-a-conda-environment) with your custom package versions. Please email us at [jupyterteam@ucdavis.edu](mailto:jupyterteam@ucdavis.edu) if you have concerns or issues with the new environment.
+
+## How do I access RStudio?
+
+To open the RStudio interface, you can select the RStudio notebook icon from the Jupyter Launcher (Ctrl+Shift+L). This will open a new JupyterLab tab which has an RStudio interface and access to all the same packages you would find elsewhere on your account.
+
+![RStudio Launcher icon](RStudio-launcher.png)
+
+## How can I install custom packages?
+
+You are free to install any additional packages available through conda by [creating a conda environment](#how-do-i-create-a-conda-environment), or by running [conda install 'package'](https://docs.conda.io/projects/conda/en/latest/user-guide/concepts/installing-with-conda.html) commands within your custom environment. These packages will persist for you. You may also use `conda install` directly in the `(notebook)` environment provided by default, but these packages won't be permanent and will be wiped anytime your server restarts.
```

Comment:
You need to be careful with the word "environment" as both jupyterhub and conda have the concept "environment" and they are different things. This sentence mixes the two meanings.

Reply:
Probably should always write "conda environment" and "jupyterhub environment" for clarity. Defining them each when first used would be good too.

Reply:
Add a brief description for what our jupyterhub environments actually are (docker images), and will specify throughout the rest of document which type of environment is being talked about